ui-tests/cypress/support/utils.js

```diff
@@ -5,6 +5,10 @@ import RandExp        from 'randexp';
 // const emailRegex = new RandExp(/(?=^[^@]{2,64}@[^@]{5,255}$)^([a-z][a-z0-9]*([-._][a-z0-9]+)*)@(([a-z][-a-z0-9]+([-.][a-z0-9]+)*\.[a-z0-9]{2,4})|(((25[0-5])|(2[0-4][0-9])|(1?[1-9]?[0-9])\.){3}((25[0-5])|(2[0-4][0-9])|(1?[1-9]?[0-9]))))$/);
 const emailRegex = new RandExp(/^([a-z][a-z0-9]{1,9}([-._][a-z0-9]{1,9}){0,5})@(([a-z][-a-z0-9]{4}([-.][a-z0-9]{1,9}){0,10}\.[a-z0-9]{2,4})|((((25[0-5])|(2[0-4][0-9])|(1?[1-9]?[0-9]))\.){3}((25[0-5])|(2[0-4][0-9])|(1?[1-9]?[0-9]))))$/);
 
+export const getArrayRandomElement = (array) => {
```

Comment:
This might be useful somewhere else, it's better to move it where it can be re-used